corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -177,6 +178,8 @@ def indicators(self):
                 ret.append(make_form_question_indicator(
                     prop['source'], prop['column_id']
                 ))
+            elif prop['type'] == 'case_property' and prop['source'] == 'computed/owner_name':
```

Comment:
Agreed this isn't the cleanest. Will definitely revisit it if it gets any bigger.